src/Month.jsx

```diff
@@ -121,6 +121,7 @@ let MonthView = React.createClass({
         { weeks.map((week, idx) =>
             this.renderWeek(week, idx, measure && this._renderMeasureRows))
         }
+        { this.props.popup && this.state.overlay?this._renderOverlay():null}
```

Comment:
hey there. This is fine generally, however the added check for `state.overlay` isn't needed overlay visibility is handled by the `show` prop, its not a big deal now, but if there was a close animation it would be cut off.

also can you remove the `? : null` bit its fine to keep it how it was.